From 7d3c0a2d27252109f61fef0f954d53a89704dde1 Mon Sep 17 00:00:00 2001
From: Bixuan Cui <cuibixuan@huawei.com>
Date: Fri, 3 Dec 2021 18:15:47 +0800
Subject: [PATCH] printk: export log_buf_addr_get/log_buf_len_get
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 7d3c0a2d27252109f61fef0f954d53a89704dde1


ascend inclusion
category: feature
bugzilla: NA
CVE: NA

-------------------------------------------------

Export log_buf_addr_get()/log_buf_len_get() for bbox driver.

Signed-off-by: Bixuan Cui <cuibixuan@huawei.com>
Reviewed-by: Hanjun Guo <guohanjun@huawei.com>
Signed-off-by: Yang Yingliang <yangyingliang@huawei.com>
Reviewed-by: Weilong Chen <chenweilong@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 kernel/printk/printk.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kernel/printk/printk.c b/kernel/printk/printk.c
index 16cb3837b3a5..e237ac1a6533 100644
--- a/kernel/printk/printk.c
+++ b/kernel/printk/printk.c
@@ -457,12 +457,14 @@ char *log_buf_addr_get(void)
 {
 	return log_buf;
 }
+EXPORT_SYMBOL_GPL(log_buf_addr_get);
 
 /* Return log buffer size */
 u32 log_buf_len_get(void)
 {
 	return log_buf_len;
 }
+EXPORT_SYMBOL_GPL(log_buf_len_get);
 
 /*
  * Define how much of the log buffer we could take at maximum. The value
-- 
2.26.2

